packages/vite-plugin-kit-routes/src/test/ROUTES_format-route-slash.ts

```diff
@@ -0,0 +1,169 @@
+/** 
+ * This file was generated by 'vite-plugin-kit-routes'
+ * 
+ *      >> DO NOT EDIT THIS FILE MANUALLY <<
+ */
+
+/**
+ * PAGES
+ */
+const PAGES = {
+  "/": `/`,
+  "/subGroup": `/subGroup`,
+  "/subGroup2": `/subGroup2`,
+  "/contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract`
+  },
+  "/contract/[id]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract/${params.id}`
+  },
+  "/gp/one": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/gp/one`
+  },
+  "/gp/two": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/gp/two`
+  },
+  "/main": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/main`
+  },
+  "/match/[id=int]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/match/${params.id}`
+  },
+  "/site": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site`
+  },
+  "/site/[id]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site/${params.id}`
+  },
+  "/site_contract/[siteId]-[contractId]": (params: { siteId: (string | number), contractId: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site_contract/${params.siteId}-${params.contractId}`
+  },
+  "/a/[...rest]/z": (params: { rest: (string | number)[] }) => {
+    return `/a/${params.rest?.join('/')}/z`
+  },
+  "/lay/normal": `/lay/normal`,
+  "/lay/root-layout": `/lay/root-layout`,
+  "/lay/skip": `/lay/skip`
+}
+
+/**
+ * SERVERS
+ */
+const SERVERS = {
+  "GET /contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract`
+  },
+  "POST /contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract`
+  },
+  "GET /site": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site`
+  },
+  "GET /api/graphql": `/api/graphql`,
+  "POST /api/graphql": `/api/graphql`
+}
```

Comment:
`method /the/Path`, how do you feel?

Reply:
I like it, it should feel natural to everyone, reminds me of how Swagger docs are generated/displayed (e.g. https://petstore.swagger.io/)